### PR TITLE
Push notifications bugfixes

### PR DESCRIPTION
**Pull request by lynx56**
Fixed Bugs:

- Show disabled state after not allowing on welcome screen (#8693ygf1h)
- Crash on add delegation (#86941tu1b)
- Filter networks for notifications based on chains.json (#86941vmw7)
- Blank push notification settings screen (#86941y5hk)
- Update disabled state as to layouts (#8693ygf1n)
- Disable notifications if all notification types are off (#86940awnr)
- Save becomes Active when go to settings menu and get back (#86940akjb)

Bonus:

- Enable all chain settings if chain was selected in governance
- Show push settings screen after setup

### CODE REVIEW

**Comment by stepanLav** [comment]
Task linked: [CU-8693ygf1h iOS. Show disabled state after not allowing on welcome screen](https://app.clickup.com/t/8693ygf1h)

**Comment by stepanLav** [comment]
Task linked: [CU-86940akjb iOS. Save becomes Active when go to settings menu and get back](https://app.clickup.com/t/86940akjb)

**Comment by stepanLav** [comment]
Task linked: [CU-8693ygf1n iOS. Update disabled state as to layouts](https://app.clickup.com/t/8693ygf1n)

**Comment by stepanLav** [comment]
Task linked: [CU-86941tu1b iOS. Crash on add delegation](https://app.clickup.com/t/86941tu1b)

**Comment by stepanLav** [comment]
Task linked: [CU-86940awnr iOS. Disable notifications if all notification types are off](https://app.clickup.com/t/86940awnr)

**Comment by stepanLav** [comment]
Task linked: [CU-86941vmw7 iOS. Filter networks for notifications based on chains.json](https://app.clickup.com/t/86941vmw7)

**Comment by stepanLav** [comment]
Task linked: [CU-86941y5hk iOS. Blank push notification settings screen](https://app.clickup.com/t/86941y5hk)

File: novawallet/Modules/Notifications/Management/NotificationsManagementPresenter.swift [hunk 1]
```diff
@@ -18,9 +18,11 @@ final class NotificationsManagementPresenter {
     private var modifiedTopicsSettings: PushNotification.TopicSettings?
 
     private var isSaveAvailable: Bool {
-        (settings != modifiedSettings) ||
+        let settingsWasModified = (settings != modifiedSettings) ||
             (notificationsEnabled != modifiedNotificationsEnabled) ||
             (topicsSettings != modifiedTopicsSettings)
+        let isAmbiguousState = areAllNotificationsOff() == true && modifiedNotificationsEnabled == true
```
Comment (ERussel) [inline novawallet/Modules/Notifications/Management/NotificationsManagementPresenter.swift:24]: Why do we need to check ambiguousState if we always check whether to set modifiedNotificationsEnabled to false when anything changes?

File: novawallet/Modules/Settings/SettingsWireframe.swift [hunk 2]
```diff
@@ -105,7 +105,12 @@ final class SettingsWireframe: SettingsWireframeProtocol, AuthorizationPresentab
     }
 
     func showSetupNotifications(from view: ControllerBackedProtocol?) {
-        guard let setupNotificationsView = NotificationsSetupViewFactory.createView() else {
+        let completion: () -> Void = { [weak self] in
+            self?.showManageNotifications(from: view)
```
Comment (ERussel) [inline novawallet/Modules/Settings/SettingsWireframe.swift:109]: Probably we need to check whether the setup completed successfully

**Review by ERussel — APPROVED** [review]

**Pull request merged**